docs/posts/ibis-version-v8-release/index.qmd

```diff
@@ -0,0 +1,238 @@
+---
+title: "Ibis v8: streaming and more!"
+author: "Ibis team"
+date: "2024-02-05"
+categories:
+    - release
+    - blog
+---
+
+## Overview
+
+Ibis v8 marks the first release of data streaming backends in Ibis! We're
+excited to begin bridging the gap between batch and streaming systems, starting
+with [Apache Flink](https://flink.apache.org/) and
+[RisingWave](https://risingwave.com) backends in this release.
+
+We've also added a new batch backend with [Exasol](https://exasol.com), bringing
+the total number of backends Ibis supports to 20+.
```

Comment:
supernit -- if we're going to say "total number" then we should provide a number, not a lower bound.
you can 100% ignore this